src/TemplateBuilder.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Staffim\SpreadsheetBuilder;
+
+use PhpOffice\PhpSpreadsheet\Reader;
+use PhpOffice\PhpSpreadsheet\RichText\RichText;
+use PhpOffice\PhpSpreadsheet\Writer;
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+
+class TemplateBuilder
+{
+    protected const START_ROW = 0;
+    protected const START_COLUMN = 'A';
+
+    /**
+     * @var Spreadsheet
+     */
+    protected $template;
+
+    /**
+     * @var array
+     */
+    protected $rowErrors = [];
+
+    public function __construct(string $template)
```

Comment:
need to add ability to set range for template instead of `START_ROW` and `START_COLUMN` constants

Reply:
added